src/TestStack.BDDfy/Processors/UnusedExampleException.cs

```diff
@@ -1,21 +1,12 @@
 ﻿namespace TestStack.BDDfy.Processors
 {
     using System;
-    using System.Runtime.Serialization;
 
-    [Serializable]
```

Comment:
OK, I see what you mean. But then I would still need an #if for the serializable constructor?